Sources/XcodeGenCLI/XcodeGenCLI.swift

```diff
@@ -15,7 +26,9 @@ public class XcodeGenCLI {
             description: "Generates Xcode projects",
             commands: [generateCommand]
         )
-        cli.parser = Parser(router: CommandRouter(defaultCommand: generateCommand))
+        let manipulator = Manipulator(commandName: generateCommand.name)
+        cli.argumentListManipulators.insert(manipulator, at: 0)
+        cli.parser.routeBehavior = .searchWithFallback(generateCommand)
```

Comment:
Does this new routeBehaviour replace the need for the `Manipulator`? The purpose of that was to make `generate` the default command

Reply:
Looks like it works fine with just the routeBehaviour

Reply:
I tried getting rid of any custom manipulators. However, behavior is changed. So it is needed.

```console
$ swift run xcodegen # works fine
$ swift run xcodegen generate # behaviors are changed.

Usage: xcodegen [options]

Generate an Xcode project from a spec

Options:
  --cache-path <value>     Where the cache file will be loaded from and save to. Defaults to ~/.xcodegen/cache/{SPEC_PATH_HASH}
  -c, --use-cache          Use a cache for the xcodegen spec. This will prevent unnecessarily generating the project if nothing has changed
  -h, --help               Show help information
  -p, --project <value>    The path to the directory where the project should be generated. Defaults to the directory the spec is in. The filename is defined in the project spec
  -q, --quiet              Suppress all informational and success output
  -s, --spec <value>       The path to the project spec file. Defaults to project.yml

Error: command requires exactly 0 arguments
```

Reply:
I tried again, it will works as you said. Manipulators are not needed.